assets/styl/_elements/_modal.styl

```diff
@@ -20,14 +20,15 @@
         display none
 
     &-overlay
-        position fixed
-        top 0
+        position absolute
+        top -30%
```

Comment:
Why those negative value to position the overlay ?

Reply:
To avoid overlay missing under address bar for mobil when address bar disappear.